Sources/SafeDICore/Extensions/CollectionExtensions.swift

```diff
@@ -21,37 +21,51 @@
 import SwiftSyntax
 import SwiftSyntaxBuilder
 
-extension Array where Element == Dependency {
-
-    var removingDuplicateInitializerArguments: Self {
```

Comment:
This wasn't necessary. I think I wrote this very early on before dramatically modifying the shape of the DI system.